securitybot/bot.py

```diff
@@ -326,6 +372,9 @@ def alert_user(self, user, task):
         '''
         # Format the reason to be indented
         reason = '\n'.join(['>' + s for s in task.reason.split('\n')])
+        url = task.url
+        if url and url != 'N/A':
```

Comment:
yup, the default was 'N/A' so I tried to leave it that way. But since we already handle the None value at the message, it makes sense to replace N/A for None